src/Components/MovieCard/MovieCard.js

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
-import './MovieCard.css'
+import './MovieCard.css';
+import { Link } from 'react-router-dom';
 
-const MovieCard = ({ title, poster_path, average_rating }) => {
+const MovieCard = ({ movie }) => {
   return(
     <div>
-      <h2>{title}</h2>
-      <img src={poster_path}></img>
-      <p>{average_rating}</p>
+    <Link to={`/movies/${movie.id}`}>
+      <h2>{movie.title}</h2>
+      <img src={movie.poster_path}></img>
```

Comment:
Add in an alt prop or aria label with the movie title plus  "image"

Reply:
Done!